common/frame.go

```diff
@@ -1107,6 +1107,10 @@ func (f *Frame) IsVisible(selector string, opts goja.Value) bool {
 
 // ID returns the frame id
 func (f *Frame) ID() string {
+	if f == nil {
```

Comment:
Wouldn't all of these `if f == nil` checks just mask some more serious issue where the existance of frame is expected and the frame isn't actually there? It feels to me like the handling of a `frame == nil` should be done before making a call to `Frame` methods.

Reply:
They are mostly used for logging but sure I've removed them in 88b2ba9.